scripts/ek-openssl-verify

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
```

Comment:
Can you add maybe also a reference to this script in the documentation? Thanks for adding an example!